README.md

```diff
@@ -1,31 +1,118 @@
-Welcome to the KubeVirt Documentation. This welcome page is provided as
-the entrypoint to the different topics of this user-guide.
+# KubeVirt User-Guide
 
-Try it out
-==========
+[![Netlify Status](https://api.netlify.com/api/v1/badges/2430a4f6-4a28-4e60-853d-f0cc395e13bb/deploy-status)](https://app.netlify.com/sites/kubevirt-user-guide/deploys)
 
--   An easy to use demo: <https://github.com/kubevirt/demo>
+## Contributing contents
 
-Getting help
-============
+We more than welcome contributions to KubeVirt documentation. Please reach out if you happen to have an idea or find an issue with our contents!
 
--   File a bug: <https://github.com/kubevirt/kubevirt/issues>
+### Get started
 
--   Mailing list: <https://groups.google.com/forum/#!forum/kubevirt-dev>
+- Create fork of GitHub user-guide repo
 
--   Slack: <https://kubernetes.slack.com/messages/virtualization>
+- Clone repository, check out source branch and prepare the Jekyll site
+  ```console
+  git clone -b master https://github.com/mygithubname/user-guide.git && cd user-guide
+  ```
 
-Developer
-=========
+- Set up your git repo remotes like the following:
+```
+$ git remote -v
+origin	git@github.com:mygithubname/user-guide.git (fetch)
+origin	git@github.com:mygithubname/user-guide.git (push)
+upstream	git@github.com:kubevirt/user-guide.git (fetch)
+upstream	git@github.com:kubevirt/user-guide.git (push)
+```
 
+<<<<<<< HEAD
 -   Start contributing: [Appendix/Contributing](appendix/contributing.md)
+=======
```

Comment:
This seems to be a leftover from an unresolved conflict that should be cleaned up